### PR TITLE
feat: add Context class and property methods into the game_controller.py

### PR DESCRIPTION
**Pull request by asternotus**

### CODE REVIEW

File: context.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+
+
+class Context(ABC):
+    @abstractmethod
+    @property
```
Comment (elias-gvin) [inline context.py:5]: Please, change order of decorators:
property
abstractmethod
instead of:
abstractmethod
property

File: game_controller.py [hunk 2]
```diff
@@ -21,6 +22,18 @@ def __init__(self, citizens_dict: Dict[str, Card],
 
         self.game: Game
 
+    @property
```
Comment (elias-gvin) [inline game_controller.py]: i'ts better to add Context properties implementation to the end of game_controller file.
And, please, separate these properties from other game_controller methods using text below:
\# =================================================================
\# Context implementation
\# =================================================================

**Review by elias-gvin — CHANGES_REQUESTED** [review]

File: context.py (hunk 1 above)
Comment (asternotus) [reply context.py:5]: Ok, ready

File: game_controller.py (hunk 2 above)
Comment (asternotus) [reply game_controller.py]: Ok, ready

**Review by elias-gvin — APPROVED** [review]

**Pull request merged**